rtf_vis_tool/src/tests/DivNode.test.js

```diff
@@ -0,0 +1,58 @@
+/* Unit Tests for the DivNode.js Component
+
+Author: Adi Singh
+*/
+import React from 'react'
+
+// Third-Party Package Imports.
+import Enzyme, { shallow } from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+
+// Local Imports.
+import DivNode from '../Components/DivNode';
```

Comment:
is DivNode outdated now?

Is there any way to run the unit tests inside our CI? what's the one-liner?

Reply:
Yes, DivNode is outdated now. I have updated the tests with the new, correct names. To run the unit tests, just run `npm test` within the `gtsfm/rtf_vis_tool` directory.